recipes/spatial_tda/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "spatial_tda" %}
+{% set version = "0.1.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/spatial_tda-{{ version }}.tar.gz
+  sha256: f04de72963f3ff8f8139e2cd9a4ce5ddae4b9ef75a50759c629a0a2797d47923
+
+build:
```

Comment:
This looks like it should probably be `noarch: python`:
https://conda-forge.org/docs/maintainer/knowledge_base/#noarch-python

If so, please update accordingly.

Additionally, it looks like there is a python entry point, so please add it as in:
https://docs.conda.io/projects/conda-build/en/stable/resources/define-metadata.html#python-entry-points

and add a test like in:
https://docs.conda.io/projects/conda-build/en/stable/resources/define-metadata.html#test-commands